common/thorhelper/thorcommon.cpp

```diff
@@ -1074,16 +1074,20 @@ class CRowStreamReader : public CSimpleInterfaceOf<IExtRowStream>
         }
         return false;
     }
-    const byte *getNextPrefetchRow()
+    const byte *getNextPrefetchRow(size32_t & size)
     {
         while (true)
         {
             ++progress;
             if (checkEmptyRow())
+            {
+                size = 0;
                 return nullptr;
```

Comment:
trivial: clearer if break; and fell through to common ending that does same? [as if (checkExitConditions()) does]